jenkins/job-configs/kubernetes-jenkins-pull/kubernetes-pull.yaml

```diff
@@ -312,14 +312,23 @@
           max-total: 12
           trigger-phrase: 'e2e\s+test'
           cmd: |
-            export KUBE_SKIP_PUSH_GCS=y
-            export KUBE_RUN_FROM_OUTPUT=y
-            export KUBE_FASTBUILD=true
-            ./hack/jenkins/build.sh
             if [[ "${{ghprbTargetBranch:-}}" == "release-1.0" || "${{ghprbTargetBranch:-}}" == "release-1.1" ]]; then
               echo "PR GKE job disabled for legacy branches."
               exit
             fi
+
+            export KUBE_SKIP_PUSH_GCS=n
+            export KUBE_GCS_RELEASE_BUCKET=kubernetes-release-pull
+            export KUBE_RUN_FROM_OUTPUT=y
+            export KUBE_FASTBUILD=true
+
+            ./hack/jenkins/build.sh
+
+            version=$(source build/util.sh && echo $(kube::release::semantic_version))
+            gsutil -m cp -r "gs://kubernetes-release-pull/ci/${{version}}" "gs://kubernetes-release-dev/ci/${{version}}"
```

Comment:
Two questions:
1. Are you sure this won't clobber existing versions created by kubernetes-build? You may want to add a suffix to the version name.
2. If you aren't changing the version name when you copy, why not set `KUBE_GCS_RELEASE_BUCKET=kubernetes-release-dev` ?